.golangci.yml

```diff
@@ -28,12 +28,15 @@ linters-settings:
       - name: unused-parameter
       - name: var-declaration
       - name: var-naming
+  gocyclo:
+    min-complexity: 15
```

Comment:
what is 'min-complexity' is that report anything over that amount ?

Reply:
Yep